src/controller.rs

```diff
@@ -259,6 +259,9 @@ fn dispatch_editor_request(request: EditorRequest, mut ctx: &mut Context) {
         request::DocumentSymbolRequest::METHOD => {
             document_symbol::text_document_document_symbol(meta, &mut ctx);
         }
+        "kak-lsp/next-or-previous-symbol" => {
+            kak_lsp_custom::document_next_or_prev_symbol_request(meta, params, &mut ctx);
```

Comment:
I believe that the names of these commands should correspond to the method name 1:1 (I didn't check if we have exceptions).
For example, `text_document_document_symbol` corresponds to `textDocument/documentSymbol`.

The consistent name here would be `kak_lsp_next_or_previous_symbol`. Maybe we should make exceptions for kak_lsp-prefixed ones..

Reply:
I leave the renaming of the functions to what you feel appropriate. Kept the name the same for now...